rspace/src/main/scala/coop/rchain/rspace/package.scala

```diff
@@ -331,4 +332,15 @@ package object rspace {
         }
       }
     }
+
+  /** Gets a checkpoint.  Yes, this is a handwave.
+    *
+    * @param store A store which satisfies the [[IStore]] interface.
+    * @tparam C A type representing a channel
+    * @tparam P A type representing a pattern
+    * @tparam A A type representing a piece of data
+    * @tparam K A type representing a continuation
+    * @return A BLAKE2b256 representing the checkpoint
+    */
+  def getCheckpoint[C, P, A, K](store: IStore[C, P, A, K]): Blake2b256Hash = store.getCheckpoint()
```

Comment:
Wait. What's the reason for existence of this method?

Reply:
I know, seems a bit redundant?  

From a public API perspective, so far we haven't had clients operate directly on the store object, other than to close it.   I wanted the public API functions for history and rollback to follow this convention until such time as we choose to refactor.